### PR TITLE
Update cleanupregistration output for more understaing

### PR DESCRIPTION
**Pull request by quroom**
This code gives more details for cleanupregistration.
Without this, we don't know there is cleaned user or not.
We can only know, command was run or not.

### CODE REVIEW

**Comment by quroom** [comment]
Fix line error.
Let's see.
If there is more problems, I will look into laster.
Have a nice day.

**Comment by quroom** [comment]
Add new line.
By the way, is there any command to check line in win system?
I can only check invoke test and build.
Any lint test command?

File: registration/models.py [hunk 1]
```diff
@@ -274,16 +274,19 @@ def delete_expired_users(self):
...
             try:
                 if profile.activation_key_expired():
                     user = profile.user
                     logger.warning('Deleting expired Registration profile {} and user {}.'.format(profile, user))
                     profile.delete()
                     user.delete()
+                    deleted_count += 1
             except UserModel().DoesNotExist:
                 logger.warning('Deleting expired Registration profile {}'.format(profile))
                 profile.delete()
```
Comment (joshblum) [inline registration/models.py:288]: i think you should increment the count here as well since it results in a profile being deleted

Comment (quroom) [reply registration/models.py:288]: I missed it I will send pr again.

**Review by joshblum — APPROVED** [review]

**Comment by quroom** [comment]
Thanks for review.
Have a nice year.

**Pull request merged**